docs/DEVELOPMENT.md

```diff
@@ -0,0 +1,81 @@
+# Development
+
+### Requirements
+
+* Python 3
+
+### OS Dependencies
+
+* Postgresql
+
+**NOTE:** Ensure after installation the `postgres` role exists
+
+```
+> psql postgres
+Type "help" for help.
+
+postgres=# \du
+                                     List of roles
+   Role name   |                         Attributes                         | Member of
+---------------+------------------------------------------------------------+-----------
+ postgres      | Superuser, Create role, Create DB                          | {}
+```
+
+If it doesn't exist you will need to create one.
+```
+# For Mac users (prepend sudo if necessary):
+createuser -s postgres
+```
+
+* libxml2
+
+```bash
+# For Mac users with brew:
+brew unlink libxml2
+brew uninstall libxml2 --ignore-dependencies --force
+brew install --with-python libxml2
+brew link libxml2 --force
+```
+
+## Setup
+
+* Install virtual environment
+
+```
+/usr/bin/python3 -m venv venv
+venv/bin/pip install -U setuptools pip wheel
+```
+
+* Install application dependencies
+
+```
+venv/bin/pip install -r requirements/base.txt
+```
+
+* Collect static files
+
+```
+venv/bin/python manage.py collectstatic
+```
+
+* Create database
+
+```
+> psql postgres
+postgres=# CREATE DATABASE fee_calculator WITH OWNER postgres ENCODING 'utf-8';
+CREATE DATABASE
+```
+
+* Migrate database
+
+```
+venv/bin/python manage.py migrate --no-input
+```
+
+* Run the server
+
+By default bounds to port 8080
+
+```
+DEBUG="True" venv/bin/uwsgi --ini uwsgi.ini
```

Comment:
For development, I would run with `python manage.py runserver 8000` instead, as this will auto-reload code changes and serve static files directly (don't need to do `collectstatic` in this case).